Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -40,6 +40,35 @@
 
 namespace PhysX
 {
+    void EditorProxyCylinderShapeConfig::Reflect(AZ::ReflectContext* context)
+    {
+        if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
```

Comment:
suggestion: Not required but I've seen feedback in the past where for this people prefer `auto*` rather than just `auto` to indicate it's a pointer (I don't mind here as the type is on the right anyway but just wanted to mention it as it often gets pointed out).

Reply:
Yeah I prefer `auto*` too, updated, thanks!